onnxscript/rewriter/no_op.py

```diff
@@ -23,6 +23,14 @@ def div_by_1(op, x):
     return x / 1
 
 
+def dropout_zero(op, x):
+    return op.Dropout(x, ratio=0.0, _allow_other_attributes=True)
```

Comment:
@gramalingam will `ratio=0` match `ratio=0.0`?

Reply:
Hmm, it's a good question what it *should* do. Currently, I think this test will be performed as [here](https://github.com/microsoft/onnxscript/blob/74ae4cc0efac8a33869a4a0dfd520dd42702c726/onnxscript/rewriter/pattern.py#L118) ... via `attr.value == 0`, so I guess it will succeed regardless of whether the attribute is an int or float? If we add a type-check in future, it may not.

Reply:
Sounds good thanks! I think it should match both like it currently does.

Reply:
BTW, I checked and noticed that the the allow_other_attributes' default value is True, so it doesn't have to be specified. (Maybe the original issue was unrelated to this, it should have worked even without this flag.)

Reply:
Thanks, updated